src/pages/heeyeolKim/Login/Login.js

```diff
@@ -1,109 +1,105 @@
-            });
-        } else if (e.target.type === "password") {
-            this.setState({
-                pwValue: e.target.value
-            });
-        }
-    }
 
     goToMain = () => {
-        if (this.state.buttonToGo === true) {
-            this.props.history.push('/main-heeyeol') // 정확히 어떤 의미?  
-        } // 아니면 그냥 초기값(false이므로 else를 작성 안해도 됨)
+        fetch('http://10.58.5.70:8000/user/login', 
+        {method: 'POST', 
+        body: JSON.stringify({
+            email: this.state.idValue,
+            password: this.state.pwValue,
+        })
+        })
+        .then((res) => {
+            console.log(res);
+            return res.json();})
+        .then((data) => {
+            console.log("결과: ", data);
+            if (data.MESSAGE === "SUCCESS") {
+                this.props.history.push('/main-heeyeol')
+                } else {
+                    alert("실패!");
+                }
+        });
     }
 
     render() {
         return (
             <div className = "Login">
                 <main className="mainSection">
                     <section className="leftImage" >
-                        <img src={loginPageImage} alt="#" />
+                        <img src={loginPageImage} alt="사진을 찾을 수 없습니다." />
                     </section>
                 
                     <section className="rightLoginPage">
                         <section className="logIn">
                             <h1>
-                                <img src={logoImage} alt="" />
+                                <img src={logoImage} alt="사진을 찾을 수 없습니다." />
                             </h1>
                             <div className="logInMain" onKeyUp={this.handleActivation}>
                                 <input 
                                 className="changeColorId" 
+                                name = "email"
                                 type="text" 
                                 placeholder="전화번호, 사용자 이름 또는 아이디"
                                 onChange = {this.handleInput}
                                 />
                                 <input 
-                                className="changeColorPw" 
+                                className="changeColorPw"
+                                name = "password"
                                 type="password" 
                                 placeholder="비밀번호"
                                 onChange = {this.handleInput}
                                 />
                                 <button 
                                 style={{opacity: this.state.buttonOpacity}}
-                                className="logInButton"
-                                onClick={this.goToMain}
-                                >
-                                    <div>
-                                        로그인
-                                    </div>
+                                className={this.state.buttonToGo}
```

Comment:
```suggestion
                                className={this.state.buttonToGo ? "buttonOn" : "buttonOff"}
```
- state.buttonToGo 데이터를 Boolean으로 관리해준다면 위와 같이 할 수 있겠네요!